java/src/apps/Apps.java

```diff
@@ -263,7 +264,13 @@ public boolean execute() {
         if (file.exists()) {
             log.debug("start load config file {}", file.getPath());
             try {
-                configOK = InstanceManager.getOptionalDefault(jmri.ConfigureManager.class).load(file, true);
+                ConfigureManager cmOD = InstanceManager.getOptionalDefault(jmri.ConfigureManager.class);
+                if (cmOD != null) {
+                    configOK = cmOD.load(file, true);
```

Comment:
Back on line 213, a configure manager was installed.  That means that getDefault should work without complaining about a missing default.  (And getOptionalDefault will never return null) Would this and similar code be cleaner if it just used "getDefault" on the one line form?
````
configOK = InstanceManager.getDefault(jmri.ConfigureManager.class).load(file, true);
````